jdk/src/share/classes/java/util/concurrent/ThreadLocalRandom.java

```diff
@@ -133,22 +133,7 @@
     /**
      * The next seed for default constructors.
      */
-    private static final AtomicLong seeder = new AtomicLong(initialSeed());
-
-    private static long initialSeed() {
-        String pp = java.security.AccessController.doPrivileged(
-                new sun.security.action.GetPropertyAction(
-                        "java.util.secureRandomSeed"));
-        if (pp != null && pp.equalsIgnoreCase("true")) {
-            byte[] seedBytes = java.security.SecureRandom.getSeed(8);
-            long s = (long)(seedBytes[0]) & 0xffL;
-            for (int i = 1; i < 8; ++i)
-                s = (s << 8) | ((long)(seedBytes[i]) & 0xffL);
-            return s;
-        }
-        return (mix64(System.currentTimeMillis()) ^
-                mix64(System.nanoTime()));
-    }
+    private static final AtomicLong seeder = new AtomicLong(8270523877573429709L);
```

Comment:
Where does this number come from?

Reply:
Pulled it out of my... system clock one afternoon :-).